bin/cli

```diff
@@ -34,88 +34,94 @@ function loadJSON(path) {
   return json || null;
 }
 
-var npm_major_version = exec('npm -v').toString().match(/^(\d)\./)[1];
-
-argv = optimist
-  .usage('Ensure package.json and npm-shrinkwrap.json are in sync')
-  .alias('d', 'dev')
-  .alias('h', 'help')
-  .alias('3', 'v3')
-  .describe('3', 'Perform check taking npm3 flat structure into account.')
-  .describe('d', 'Check devDependencies.')
-  .describe('h', 'Show this help message.')
-  .argv;
-
-if (argv.help) {
-  optimist.showHelp();
-  process.exit(0);
-}
-
-packageJSON = loadJSON(process.cwd() + '/package.json');
-shrinkwrapJSON = loadJSON(process.cwd() + '/npm-shrinkwrap.json');
+exec('npm -v', undefined, function (err, stdout) {
```

Comment:
From [the docs](https://nodejs.org/docs/latest-v0.10.x/api/child_process.html#child_process_child_process_exec_command_options_callback), I think you can omit the `undefined` value and pass two params: the command and the callback.